qiskit/algorithms/aux_ops_evaluator.py

```diff
@@ -16,7 +16,7 @@
 import numpy as np
 
 from qiskit import QuantumCircuit
-from qiskit.algorithms.eigen_solvers.eigen_solver import ListOrDict
+from qiskit.algorithms.minimum_eigen_solvers.minimum_eigen_solver import ListOrDict
```

Comment:
Shouldn't this import directly from algorithms rather than indirectly from some other code (minimum_eigen_solver) that imports it?

Reply:
It should!

```suggestion
from .list_or_dict import ListOrDict
```